e2e/page-objects/profilePage.ts

```diff
@@ -67,6 +67,27 @@ export class ProfilePage extends HelperBase {
     return fullNameValue;
   }
 
+  async getOsmEditorValue() {
+    const osmEditorValue = await this.page
+      .getByText('Default OpenStreetMap editor (beta)')
+      .locator('..')
+      .locator('span')
+      .textContent();
+    return osmEditorValue;
+  }
+
+  async setOsmEditorValue(osmEditorValue: string) {
+    await this.page
+      .getByText('Default OpenStreetMap editor (beta)')
+      .locator('..')
+      .locator('..')
```

Comment:
using many xpath locators ('..') is not really reliable. They can break after several FE changes. There is a way to work with lists in Playwright. Check https://playwright.dev/docs/locators#lists out. 

What elements we need to work through
![image](https://github.com/user-attachments/assets/9d80b15c-b39b-487a-a556-3af527dcb581)

Reply:
to stop JS execution and to see the picture above use `setTimeout(() => {debugger}, "3000")` script written in the console